### PR TITLE
Feature/crud user

### PR DESCRIPTION
**Pull request by benKazmierczak**
Adding a customize form with errors message and materialize style

### CODE REVIEW

**Review by Backins — APPROVED** [review]
Test and approve

**Review by mehdou92 — APPROVED** [review]

File: .docker/php/php.ini [hunk 1]
```diff
@@ -396,7 +396,7 @@ max_input_time = 60
 
 ; Maximum amount of memory a script may consume (128MB)
 ; http://php.net/memory-limit
-memory_limit = 128M
+memory_limit = 512M
```
Comment (ivanPrunier) [inline .docker/php/php.ini:399]: 😂

File: templates/form/form_theme.html.twig [hunk 2]
```diff
@@ -0,0 +1,188 @@
+{% extends "form_div_layout.html.twig" %}
+
+{%- block form_row -%}
```
Comment (ivanPrunier) [inline templates/form/form_theme.html.twig:3]: Why are you using a dash in `{%-` and `-%}` ?

**Review by ivanPrunier — APPROVED** [review]

File: templates/form/form_theme.html.twig (hunk 2 above)
Comment (benKazmierczak) [reply templates/form/form_theme.html.twig:3]: it allows to delete spaces at the beginning and end. Here are the links of the doc if you want to check :[https://twig.symfony.com/doc/2.x/templates.html#whitespace-control](url)

**Review by jeremNoel — APPROVED** [review]

**Pull request merged**